src/content/en/tools/workbox/guides/migrations/migrate-from-v2.md

```diff
@@ -444,6 +444,13 @@ updated API surface.
 - The API now exposes two classes, `GenerateSW` and `InjectManifest`. This makes the toggling
 between modes explicit, vs. the v2 behavior where behavior changed based on the presence of `swSrc`.
 
+- By default, assets in the webpack compilation pipeline will be precached, and it is no longer
```

Comment:
This note should probably also be added to the `glob*` option documentation on https://developers.google.com/web/tools/workbox/modules/workbox-webpack-plugin - right now the `globPattern` docs seem to suggest only JS, CSS and HTML files are precached since it says that the default value is `['**/*.{js,css,html}']`

Reply:
Good point. Thanks.